src/client/java/teammates/client/scripts/DataMigrationEntityAttributes.java

```diff
@@ -0,0 +1,22 @@
+package teammates.client.scripts;
+
+import teammates.common.datatransfer.attributes.EntityAttributes;
+
+/**
+ * Base script for all data migration scripts involving {@link EntityAttributes}.
+ *
+ * @param <T> The entity attributes type to be migrated by the script.
+ */
+public abstract class DataMigrationEntityAttributes<T extends EntityAttributes<?>> extends DataMigrationBaseScript<T> {
```

Comment:
I still don't agree with that notion, because the only things that exist in the DB (and therefore can be migrated) are entities.
If anything, the difference is one can migrate the entities using attributes as helper/wrapper (which is what is being done now), or migrating the entities directly.